.gitbook/contents/hc/2_addsub/1_write-handler.md

```diff
@@ -6,15 +6,40 @@ The first step is to write our *off-chain handler*. This handler is responsible
 
 Let's begin by allowing our handler to receive two numbers. It will perform both addition and subtraction on these numbers. If the result of the subtraction results in an underflow (i.e. if `b > a` in the expression `a-b`), the handler will respond with an underflow error. Create a `Python` file in which to copy all the following code blocks.
 
-First, we'll need to import some `Python` libraries to help with our example:
+First, we'll need to import the Hybrid Compute SDK and some `Python` libraries to help with our example:
 
 ```python
+sdk = HybridComputeSDK()
+
 from web3 import Web3
 from eth_abi import abi as ethabi
-from offchain_utils import gen_response, parse_req
 ```
 
-The first two, `Web3` and `eth_abi`, are established `PyPI` libraries. The third library, `offchain_utils`, was developed by the Boba Foundation. You can view the [library](https://github.com/bobanetwork/rundler-hc/blob/boba-develop/hybrid-compute/offchain/offchain_utils.py) on our Github.
+The last two, `Web3` and `eth_abi`, are established `PyPI` libraries. The third library, `offchain_utils`, was developed by the Boba Foundation. You can view the [library](https://github.com/bobanetwork/rundler-hc/blob/boba-develop/hybrid-compute/offchain/offchain_utils.py) on our Github.
+
+Note: if writing your handler in TypeScript, import the Hybrid Compute SDK and its utility functions like this:
+
+```typescript
+import { HybridComputeSDK } from '@bobanetwork/aa-hc-sdk-server';
+
+const sdk = new HybridComputeSDK();
```

Comment:
Python + TypeScript SDK on the backend = are equal and do the same thing. They are interchangable based on the preference by the developer. 

We shouldn't mix them in one tutorial. Ideally we have something like this for "TypeScript" and "Python" when documenting the *Backend* SDK: 

![image](https://github.com/user-attachments/assets/034751f1-1e71-4199-afac-ff155afb581e)

Reply:
I didn't realize gitbook had tab functionality! I was planning on switching when we converted to docusaurus, but I can thankfully convert it now.